src/pin_slab.rs

```diff
@@ -187,7 +186,7 @@ impl<T> PinSlab<T> {
     #[inline(always)]
     unsafe fn internal_get(&mut self, key: usize) -> Option<&T> {
```

Comment:
Does this?

Reply:
Nope. Removing this and the other one also let us get rid of a couple other unsafe blocks.